src/string/utf16.cr

```diff
@@ -106,6 +106,7 @@ class String
   #
   # Invalid values are encoded using the unicode replacement char with
   # codepoint `0xfffd`.
+  @[Constructor]
```

Comment:
Um, is this a case where it's trying to compile the program using an older Crystal version that doesn't have the annotation defined? Guess the solution is to do a version check?

Reply:
Or possibly add the annotation, but not use it until a version with the annotation is released?

Reply:
Huh, but `annotation Constructor` *is* defined in stdlib. So it should compile, as does this program:
```cr
annotation Constructor
end

@[Constructor]
def foo
end

foo
```

Maybe the require order is at odds?

Reply:
Bingo. String is required before the annotations.